reactApp2/child.jsx

```diff
@@ -90,22 +65,26 @@ class Child extends React.Component {
         <ul>   
           <button 
             type="button" 
-            onClick = {this.handleActiveClick}>Active
+            onClick = {this.handleActiveClick}>displayActiveList
           </button>
           <button 
             type="button" 
-            onClick = {this.handleDeletedClick}>Deleted
-          </button>
+            onClick = {this.handleDeletedClick}>displayDeletedList
+          </button>  
           <button 
             type="button" 
-            onClick = {this.handleAllClick}>All
+            onClick = {this.handleAllClick}>displayAllList
           </button>
           <h4 style = {{color: 'green'}}>
             {this.state.state }
           </h4>
-          {this.state.filterArray.map((item,index) => 
-              <li key={index}>{item}</li>
-            )}
+            {this.state.array.map((item,index) => {
```

Comment:
@VARTIKASINGH1234 create new variable 'this.state.array' to new variable in render method.